docs/markdown/Release-notes.md

```diff
@@ -1 +1,6 @@
 # Release notes
+
+## Added Wind River Diab compiler suite for bare-metal PowerPC
+
+Tested with C++, C and assembler using [Diab](https://www.windriver.com/products/diab-compiler) 5.2.1.0 from 2004.
```

Comment:
The release notes are compiled at release time by a script from individual snippets in `docs/markdown/snippets/*.md`, so this should be moved there.